components/brave_extension/extension/brave_extension/background/api/shieldsAPI.ts

```diff
@@ -102,8 +102,9 @@ export const setAllowAds = (origin: string, setting: string) =>
  * @param {string} setting 'allow' or 'block'
  * @return a promise which resolves with the setting is set
  */
-export const setAllowTrackers = (origin: string, setting: string) =>
-  chrome.braveShields.setAdControlTypeAsync(setting, origin)
+export const setAllowTrackers = (origin: string, setting: string) => {
+  return chrome.braveShields.setAdControlTypeAsync(setting, origin)
```

Comment:
I don't think we need to return anything here

Reply:
we actually do need to return the promise here, but the `return` is implicit so could be removed